client/src/shared/components/general/RecentTransactions.js

```diff
@@ -82,7 +82,8 @@ const RecentTransactions = (props) => {
                 const responseData = await sendRequest(endpoint, 'GET', null, {
                     Authorization: 'Bearer ' + auth.token,
                 });
-                setloadedReceipts(responseData.receipts); // set the transactions data
+             
+                setloadedReceipts(responseData.results.transactions); // set the transactions data
```

Comment:
this is not your code, but if you could kindly change this to `setLoadedReceipts` in camelCase that would probably be a better naming convention :)

Reply:
ok